daemon/algod/api/server/lib/bundle_swagger_json.sh

```diff
@@ -2,13 +2,17 @@
 
 THISDIR=$(dirname $0)
 
-cat <<EOM | gofmt > $THISDIR/bundledSpecInject.go
+LICENSE_LOCATION="$THISDIR"/../../../../../scripts/LICENSE_HEADER
+LICENSE=$(sed "s/{DATE_Y}/$(date +"%Y")/" "$LICENSE_LOCATION")
+
+printf "%s\n" "$LICENSE" > $THISDIR/bundledSpecInject.go
+cat <<EOM | gofmt >> $THISDIR/bundledSpecInject.go
 // Code generated during build process, along with swagger.json. DO NOT EDIT.
```

Comment:
I guess you added the license because `check_license.sh` started failing?

It would be better to leave it out and change the comment to `Code generated by bundle_swagger_json script. DO NOT EDIT.`, or add `DO NOT EDIT` to the `check_license.sh` script. The relative path seems like something that is definitely going to break someday in the future.

Reply:
Yes, having the file permanent without having the license was causing issues.

I think that adding a comment `Code generated by bundle_swagger_json script` and adding "DO NOT EDIT", is also a good idea. Adding `Code generated by ..` should exclude it from the license check.




We already have auto-generated files that have the license header.. so I'd rather stick with the existing convention.